src/client/src/OpenFin/utils/layoutTemplates.ts

```diff
@@ -0,0 +1,338 @@
+import { ROUTES_CONFIG } from "@/constants"
+import { constructUrl } from "@/utils/url"
+
+export const mainLayout = {
+  settings: {
```

Comment:
You could store these settings in a variable and share it over all the layouts since they all seem to be the same.

Reply:
`const createStack = (overrides: Partial<StackOptions> //an example type for the stack options) => ({
  //default options,
...overrides
})` Could work to deal with any variance in the objects

Reply:
That makes sense, I will be making a common createStack() function and pass in width, height, and content as parameters.